src/css/avatar.css

```diff
@@ -0,0 +1,43 @@
+@layer components {
+    .bcc-avatar-xs {
+        @apply text-xs;
+    }
+    .bcc-avatar-sm {
+        @apply text-sm;
+    }
+    .bcc-avatar-md {
+        @apply text-base;
+    }
+    .bcc-avatar-lg {
+        @apply text-lg;
+    }
+    .bcc-avatar-xl {
+        @apply text-xl;
+    }
+    .bcc-avatar-2xl {
+        @apply text-2xl;
+    }
+
+    .bcc-avatar {
+        @apply h-[2.2857em] w-[2.2857em] rounded-full text-center font-semibold leading-[2.2857em] shadow-[0_0.071em_0.142em_rgba(0,0,0,0.05)];
```

Comment:
Because this class is including the `leading` which will affect line-height, it needs to be after the size-classes in the CSS.